ray_data_eval/simulator/policies.py

```diff
@@ -124,3 +124,71 @@ def on_task_state_change(self, task: TaskSpec, task_state: TaskState):
             self._buffer_diff[self._current_tick + task.duration] += (
                 task.output_size - task.input_size
             )
+
+
+class EqualInputOutputRateSchedulingPolicy(SchedulingPolicy):
+    """
+    A policy that schedules consumer to be the same rate as the producer
```

Comment:
be more specific: "consumer's rate of consuming input data to be the same as the producer's rate of producing output data"